charts/zesty/values.yaml

```diff
@@ -1,208 +1,903 @@
+# required: true
+# @schema
 admission:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: false
+  # @schema
+  # type: object
+  # required: true
+  # @schema
   service:
+    # @schema
+    # type: string
+    # required: true
+    # @schema
     name: "kompass-admission-controller-svc"
 
+# @schema
+# type: object
+# required: true
+# @schema
 pvcExpansion:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: true
-  # -- (string) A percentage threshold to limit pvb utilization
+  # @schema
+  # type: integer
+  # required: true
+  # minimum: 0
+  # maximum: 100
+  # @schema
+  # Percentage threshold to limit PVC utilization
   threshold: 80
-  # -- (string) The size by which to expand a PVC that exceeded its disk utilization
-  # threshold. The size can be either percentage or resource quantity ("20%"/"5Gi")
-  expansionSize: "20%"  # Amount to expand by,
-
+  # @schema
+  # type: string
+  # required: true
+  # @schema
+  # Size by which to expand a PVC that exceeded its disk utilization threshold
+  # Can be either percentage or resource quantity ("20%"/"5Gi")
+  expansionSize: "20%"
+
+# @schema
+# type: object
+# required: false
+# additionalProperties: true
+# @schema
 cxLogging:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: false
+  # @schema
+  # type: string
+  # required: false
+  # @schema
   clusterName: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
   apiKey: ""
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   domain: "eu2.coralogix.com"
+  # @schema
+  # type: string
+  # required: true
+  # format: uri
+  # @schema
   logUrl: "https://ingress.eu2.coralogix.com:443/api/v1/logs"
+  # @schema
+  # type: string
+  # required: true
+  # format: uri
+  # @schema
   timeDeltaUrl: "https://ingress.eu2.coralogix.com:443/sdk/v1/time"
+  # @schema
+  # type: string
+  # required: true
+  # format: uri
+  # @schema
   ingressLogsUrl: "https://ingress.eu2.coralogix.com/logs/v1/singles"
+  # @schema
+  # type: string
+  # required: true
+  # format: uri
+  # @schema
   ingressUrl: "https://ingress.eu2.coralogix.com/"
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   otelEndpoint: ""
 
+# @schema
+# type: object
+# required: true
+# @schema
 selfMonitoring:
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   schedule: "*/5 * * * *"
+  # @schema
+  # type: object
+  # required: false
+  # @schema
   resources:
+    # @schema
+    # type: object
+    # required: false
+    # @schema
     requests:
+      # @schema
+      # type: string
+      # required: false
+      # pattern: ^[0-9]+(\.[0-9]+)?m?$
+      # @schema
       cpu: 100m
+      # @schema
+      # type: string
+      # required: false
+      # pattern: ^[0-9]+(\.[0-9]+)?(Ki|Mi|Gi|Ti|Pi|Ei|k|M|G|T|P|E)?$
+      # @schema
       memory: 256Mi
+    # @schema
+    # type: object
+    # required: false
+    # @schema
     limits:
```

Comment:
Don't we have to set additionalProperties to true, otherwise setting a CPU limit will fail?